recipe/meta.yaml

```
@@ -18,8 +18,14 @@ requirements:
     - {{ compiler('rust') }}
     - {{ compiler('c') }}
     - {{ stdlib("c") }}
+    - python >=3.10
```

Logs indicate this links to `libpython3.12.so.1.0`.
So, we'd want `python` in `host` too.
For cross-compilation (`osx-arm64`) we'd need
```suggestion
    - python                              # [build_platform != target_platform]
    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
```
in `build` too.
I didn't check how `datavzrd` uses Python, but since it seems to link to `libpython3.12.so.1.0`, we'd not want the ` >=3.10` here but instead add `and py<310` to the `build/skip` selector to get builds that work with non-`3.12` `python`s.